bigtable/tests/instance_admin_integration_test.cc

```diff
@@ -62,6 +74,34 @@ TEST_F(InstanceAdminIntegrationTest, ListInstancesTest) {
   }
 }
 
+/// @test Verify that InstanceAdmin::DeleteInstances works as expected.
+TEST_F(InstanceAdminIntegrationTest, DeleteInstancesTest) {
+  // The emulator does not support instance operations.
+  if (UsingCloudBigtableEmulator()) {
+    return;
+  }
+  std::string instance_id = "DeleteInstancesTest_instance";
```

Comment:
no action needed here, but what we should do is create an random instance name, that way the opportunity for collision is really small.  We do something like that in the benchmarks (for table names).